packages/cli/src/__tests__/cliExecuteSwap.test.ts

```diff
@@ -4,7 +4,7 @@ import { parseArgs } from '../utils';
 const localnet = `swap
   --wallet-private-key 0x2
   --chainflip-network localnet
-  --dest-token USDC
+  --dest-asset USDC
```

Comment:
will check what we use in the bouncer and create an update pr once this is merged

Reply:
will also try to update the docs